### ISSUE TITLE
🐛:  Repeated task model import fails

### ISSUE THREAD

**Issue by bilalshaikh42** [reproducible-biomedical-modeling/CRBM-Viz#3936]
### Location

runBiosimulations

### What happened?

The following simulation run fails due to being unable to import a model component from a contained subdirectory 

https://run.biosimulations.dev/simulations/61cf83e483721ec991439924#tab=log

@jonrkarr I am not able to tell why this is not working. Is the import in the CellML not correct? 

### What made it happen?

Running a model that refrences a model contained in a subdirectory in the following method: 

```
<model cmeta:id="Weinstein_2000_HATPase" name="Weinstein_2000_HATPase" xmlns="http://www.cellml.org/cellml/1.1#" xmlns:cellml="http://www.cellml.org/cellml/1.1#" xmlns:cmeta="http://www.cellml.org/metadata/1.0#" xmlns:xlink="http://www.w3.org/1999/xlink">
    <import xlink:href="Units/Units.cellml">
        <units name="mM" units_ref="mM"/>
        <units name="per_s" units_ref="per_s"/>
        <units name="mV" units_ref="mV"/>
        <units name="umol_per_s_per_cm2" units_ref="umol_per_s_per_cm2"/>
    </import>
    <import xlink:href="Weinstein_2000_HATPase.cellml">
        <component component_ref="H_ATPase" name="H_ATPase"/>
    </import>
....
</model>
```

The folder structure of the project is as follows: 

```
├── chang_2001c.png
├── chang_2001d.png
├── documentation-example.html
├── documentation.html
├── documentation-model.html
├── HATPase_test.cellml
├── HATPase_test.sedml
├── manifest.xml
├── metadata.rdf
├── test.png
├── Units
│   └── Units.cellml
└── Weinstein_2000_HATPase.cellml
```

### Version

Development (Deployed on a .dev domain)

### Relevant log output

```shell
============================================= Executing COMBINE archive=================================
INFO:    Using cached SIF image
Traceback (most recent call last):
  File "/home/opencor/OpenCOR/bin/../python/bin/set_python_path.py", line 247, in 
    if not update_paths(args.path, args.scripts_dir, args.clear_args, args.extra_args):
  File "/home/opencor/OpenCOR/bin/../python/bin/set_python_path.py", line 213, in update_paths
    update_scripts(scripts_dir, new_path, clear_args, extra_args)
  File "/home/opencor/OpenCOR/bin/../python/bin/set_python_path.py", line 125, in update_scripts
    update_script(os.path.join(bin_dir, fn), new_path, clear_args, extra_args)
  File "/home/opencor/OpenCOR/bin/../python/bin/set_python_path.py", line 118, in update_script
    with open(script_filename, 'w') as f:
OSError: [Errno 30] Read-only file system: '/home/opencor/OpenCOR/bin/../python/bin/biosimulators-utils'
/home/opencor/OpenCOR/python/lib/python3.7/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: COMBINE/OMEX archive may be invalid.
  - The manifest does not include its parent COMBINE/OMEX archive. Manifests should include their parent COMBINE/OMEX archives.
  warnings.warn(termcolor.colored(message, Colors.warning.value), category)
/home/opencor/OpenCOR/python/lib/python3.7/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: The SED document is potentially incorrect.
  - Model `model` may be invalid.
    - The model file `HATPase_test.cellml` may be invalid.
      - Validation is not available for CellML 1.1 files.
  - Output `plot1` may be invalid.
    - Some simulation tools will not be able to generate this output because it uses data from repeated tasks. Output for repeated tasks is an experimental feature of SED-ML, which is not officially supported.
  warnings.warn(termcolor.colored(message, Colors.warning.value), category)
/home/opencor/OpenCOR/python/lib/python3.7/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: The COMBINE/OMEX archive may be invalid.
  - The SED-ML file at location `HATPase_test.sedml` may be invalid.
    - Model `model` may be invalid.
      - The model file `HATPase_test.cellml` may be invalid.
        - Validation is not available for CellML 1.1 files.
    - Output `plot1` may be invalid.
      - Some simulation tools will not be able to generate this output because it uses data from repeated tasks. Output for repeated tasks is an experimental feature of SED-ML, which is not officially supported.
  warnings.warn(termcolor.colored(message, Colors.warning.value), category)
/home/opencor/OpenCOR/python/lib/python3.7/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: The SED document is potentially incorrect.
  - Output `plot1` may be invalid.
    - Some simulation tools will not be able to generate this output because it uses data from repeated tasks. Output for repeated tasks is an experimental feature of SED-ML, which is not officially supported.
  warnings.warn(termcolor.colored(message, Colors.warning.value), category)
Archive contains 1 SED-ML documents with 1 models, 1 simulations, 2 tasks, 0 reports, and 1 plots:
  HATPase_test.sedml:
    Tasks (2):
      repeatedTask
      task1
    Plots (1):
      plot1: 1 curves

Executing SED-ML file 1: HATPase_test.sedml ...

Bundling outputs ...
Cleaning up ...

============= SUMMARY =============
Executed 1 SED documents:
  SED documents (1):
    Succeeded: 0
    Skipped: 0
    Failed: 1
  Tasks (2):
    Succeeded: 1
    Skipped: 0
    Failed: 1
  Outputs (1):
    Succeeded: 0
    Skipped: 1
    Failed: 0

The COMBINE/OMEX did not execute successfully:

  The SED document did not execute successfully:
  
    The task does not describe a valid simulation:
    
      Error: the imports could not be fully instantiated (/tmp/ imports Units/Units.cellml, which contents could not be retrieved).
srun: error: xanadu-55: task 0: Exited with exit code 1
srun: Terminating job step 3840322.1
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jonrkarr** [comment]
I'll look at this. Likely the imported files are not in the expected location.

**Comment by jonrkarr** [comment]
I haven't looked into this yet. This could be an issue with BioSimulators-utils not executing the repeated task from the intended working directory (from which paths to imported models are described).

**Comment by jonrkarr** [comment]
Working directories weren't set properly in 3 places. This is fixed in BioSimulators-utils 0.1.55 and BioSimulators-OpenCOR 0.0.13.

**Comment by jonrkarr** [comment]
The example archive now runs successfully. Example run: https://run.biosimulations.dev/simulations/61d33e6ea7318728eb5fba00
